apps/etterlatte-saksbehandling-ui/client/src/components/behandling/aktivitetsplikt/Oversikt.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Heading } from '@navikt/ds-react'
+import React from 'react'
+import { AktivitetspliktStatusTagOgGyldig } from '~shared/tags/AktivitetspliktStatusOgGyldig'
+import { useAktivitetspliktOppgaveVurdering } from '~components/aktivitetsplikt/OppgaveVurderingRoute'
+
+export const Oversikt = () => {
+  const { vurdering } = useAktivitetspliktOppgaveVurdering()
+  return (
+    <>
+      <Box paddingInline="16" paddingBlock="16 4" maxWidth="120rem">
+        <Heading level="1" size="large">
+          Oppfølging av aktivitet
+        </Heading>
+        <Heading size="medium" level="2">
+          Du skal nå vurdere brukers aktivitetsgrad
+        </Heading>
+        <p>
```

Comment:
```suggestion
        <BodyShort>
```
så setter den litt line-spacing etc. fra designsystemet

Reply:
Jess lurt